Shared/Samples/Set up location-driven geotriggers/SetUpLocationDrivenGeotriggersView.swift

```diff
@@ -0,0 +1,336 @@
+            let fenceFeature = fenceNotificationInfo.fenceGeoElement as! ArcGISFeature
+            let geotriggerName = fenceNotificationInfo.geotriggerMonitor.geotrigger.name
+            
+            // Handle notification types.
+            DispatchQueue.main.async { [weak self] in
+                guard let self = self else { return }
+                switch fenceNotificationInfo.fenceNotificationType {
+                case .entered:
+                    // The user entered a geofence: add the feature for future querying.
+                    self.featureNamesInFenceGeotrigger[geotriggerName, default: []].append(featureName)
+                    self.nearbyFeatures[featureName] = fenceFeature
+                case .exited:
+                    // The user left the geofence: remove the feature from the dicts.
+                    if let poppedFeatureName = self.featureNamesInFenceGeotrigger[geotriggerName]?.popLast() {
+                        self.nearbyFeatures.removeValue(forKey: poppedFeatureName)
+                    }
+                @unknown default:
+                    fatalError("Unexpected fence notification type.")
+                }
+                
+                // Update the status texts.
+                self.updateStatusText(featureName: featureName, notificationType: fenceNotificationInfo.fenceNotificationType)
+            }
+        }
+        
+        /// Update the status texts with the feature name and notification type.
+        /// - Parameters:
+        ///   - featureName: The name of the feature.
+        ///   - notificationType: The fence `NotificationType`.
+        private func updateStatusText(featureName: String, notificationType: FenceGeotriggerNotificationInfo.NotificationType) {
+            // Set fence geotrigger text.
+            let typeString = notificationType == .entered ? "Entered" : "Exited"
+            self.fenceGeotriggerText = String(format: "%@ the geofence of %@", typeString, featureName)
+            
+            // Set nearby features text.
+            if nearbyFeatures.isEmpty {
+                self.nearbyFeaturesText = "No nearby features."
+            } else {
+                self.nearbyFeaturesText = String(format: "Nearby: %@", ListFormatter.localizedString(byJoining: nearbyFeatures.keys.sorted()))
+            }
+        }
+        
+        /// Create and start a location display from the location data source.
+        /// - Returns: A new `LocationDisplay` object.
+        private func makeLocationDisplay() -> LocationDisplay {
+            let locationDisplay = LocationDisplay(dataSource: locationDataSource)
+            locationDisplay.autoPanMode = .recenter
+            locationDisplay.initialZoomScale = 1000
+            Task.detached { [unowned self] in
+                do {
+                    try await locationDisplay.dataSource.start()
+                } catch {
+                    self.error = error
+                }
+            }
```

Comment:
```suggestion
```
This is unnecessary. `.task(id: isShowingPopup)` will start the location display when the view is loaded.